app/controllers/api/v4/patient_controller.rb

```diff
@@ -1,4 +1,7 @@
 class Api::V4::PatientController < PatientAPIController
```

Comment:
I see the controller name is the singular form, should we disambiguate it more from a potential V4 `PatientsController`? Maybe `PatientAppController`?

Reply:
No; this is standard Rails convention for a singular vs. plural route